### PR TITLE
Edit Jadwal

### PR DESCRIPTION
**Pull request by jonathanfilbert**
Changes are live on:
http://susunjadwal.cs.ui.ac.id/
Frontend:
- Edit Jadwal page (a page just like create schedule but with prefilled schedule and agenda)
- Option to delete a schedule when a schedule's sks reaches 0 during editing a schedule
- An edit button at the view schedule page that redirects to the edit page

Backend:
- A PUT endpoint that edits the schedule_item of a schedule's payload

### CODE REVIEW

**Comment by jonathanfilbert** [comment]
Done #55 
Please review @fairuzi10 @albadrun

File: frontend/src/containers/SelectedCourses/index.js [hunk 1]
```diff
@@ -109,10 +135,11 @@ function SelectedCourses({ history }) {
           Tambah Agenda
         </Button>
         <Button
-          onClick={saveSchedule}
-          disabled={isConflict || totalCredits > 24 || schedules.length === 0}
+          onClick={() => !isEditing ? saveSchedule() : totalCredits === 0 ? handleDeleteSchedule() : updateSchedule()}
```
Comment (fairuzi10) [inline frontend/src/containers/SelectedCourses/index.js:138]: Please use `schedules.length === 0` rahter than `totalCredits === 0` like the one in line 142 so it correctly update the schedule when have no course but have agendas.

**Review by fairuzi10 — CHANGES_REQUESTED** [review]
Hi Jofil and friends, thank you for your effort in implementing this important feature!

Just a small comment, thank you! 😃

File: frontend/src/containers/SelectedCourses/index.js (hunk 1 above)
Comment (jonathanfilbert) [reply frontend/src/containers/SelectedCourses/index.js:138]: Noted. Thanks for pointing out.

Comment (jonathanfilbert) [reply frontend/src/containers/SelectedCourses/index.js]: Resolved @fairuzi10 .

**Comment by fairuzi10** [comment]
LGTM 😃

**Pull request merged**